path/__init__.py

```diff
@@ -317,13 +267,26 @@ def getcwd(cls):
         """
         return cls(getcwdu())
 
+    cwd = classmethod(getcwd)
+
+    @classmethod
+    def home(cls):
+        """ This class method works like pathlib "Path.home" class method.
+
+        :return: a Path object pointing to user home directory.
+        :rtype: Path
+        """
+        return cls(os.path.expanduser('~'))
+
     #
     # --- Operations on Path strings.
 
     def abspath(self):
         """ .. seealso:: :func:`os.path.abspath` """
         return self._next_class(self.module.abspath(self))
 
+    absolute = abspath
```

Comment:
Should the path.py project start using these methods as the canonical ones and deprecate use of the old usage?